Release40update

```diff
@@ -0,0 +1,27 @@
+### Highlights
+ 
+- We're excited to announce availability of the Public Preview of our latest Azure API Management pricing tiers: [Basic v2 and Standard v2](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview).
+- API Management is now Generally Available in the Israel Central and Italy North Azure regions.
+ 
+### New features, fixes and improvements
+ 
+- You can now specify timeout in milliseconds in the the `forward-request` policy via the new `timeout-ms` attribute.
+- We changed `validate-jwt` policy to properly handle `Bearer` and other authorization token schemas correctly in custom headers.
+- We fixed an issue where a large request log entry blocked Azure monitor log stream.
+- We fixed an issue where the `emit-metric` policy could fail the request when no diagnostic entries are configured at any of the scopes.
```

Comment:
@vladvino, I believe 
`when no diagnostic entries are configured`
should be
`when no diagnostic entries were configured` 
as this is no longer an issue now after the fix.